packages/react-api/src/Api.tsx

```diff
@@ -139,7 +147,7 @@ async function loadOnReady (api: ApiPromise, store: KeyringStore | undefined, ty
   TokenUnit.setAbbr(tokenSymbol);
 
   // finally load the keyring
-  keyring.loadAll({
+  isKeyringLoaded() || keyring.loadAll({
```

Comment:
This should always be the case. If not, we have a bug.

Reply:
That's a dirty hack to allow running more than one test. The keyring is one hell of a global variable, and there's no way to reset the damn thing. Also, as `Api` is initialised in a component, there's no easy way to have it's instance created once before all tests and reused. So that's the smallest non-breaking change that I could think of that was needed to be made to production code that allows to have more than 1 test relying on Api.